GymBro/Views/RoutineViews/RoutineSelectionView_AddRoutineSheet.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  AddRoutineView.swift
+//  GymBro
+//
+//  Created by user236772 on 4/12/23.
+//
+
+import SwiftUI
+/// A pop-up sheet in the parent view RoutineSelectionView
+///
+/// This is a page show up after clicking "+ Add Routine" button in RoutineSelectionVIew. Customer will able to type in their desire name and create a new routine in this page.
+///
+struct RoutineSelectionView_AddRoutineSheet: View {
+    
+    
+    // MARK: PROPERTY
+    
+    @Environment(\.presentationMode) var presentationMode
+    @State var textFieldText: String  = ""
+    
+    
+    // MARK: BODY
+    
+    var body: some View {
+        NavigationView {
```

Comment:
we can remove the navigationView here

Reply:
Due to it's a pop-up sheet, I think we need one more NavigationView to make sure we can see the NavigationTitle. It is the preview without adding a NavigationView:
![image](https://user-images.githubusercontent.com/44662448/232600475-76064a09-8c93-4065-89d0-01c3a57c7177.png)